examples/ios/HelloWorldSwift/BUILD

```diff
@@ -12,6 +12,40 @@ swift_library(
     data = [
         "Resources/Main.storyboard",
     ],
+    module_name = "Sources",
+)
+
+swift_library(
+    name = "Tests",
+    testonly = True,
+    srcs = [
+        "Tests/Tests.swift",
+    ],
+    deps = [
+        ":Sources",
+    ],
+)
+
+ios_unit_test(
+    name = "UnitTests",
+    minimum_os_version = "8.0",
+    deps = [":Tests"],
+)
+
+genrule(
+    name = "CreateTestCoverageManifest",
+    srcs = ["Sources/AppDelegate.swift"],
+    outs = [
+        "CoverageManifest.instrumented_files",
+    ],
+    cmd = "echo $(SRCS) > $@",
+)
+
+ios_unit_test(
+    name = "UnitTestsWithCoverageManifest",
+    minimum_os_version = "15.0",
+    test_coverage_manifest = "CoverageManifest.instrumented_files",
+    deps = [":Tests"],
```

Comment:
Coverage file from this test: https://gist.github.com/maxwellE/60bc86c1cae5730a0698a2a21a862dfe